py-polars/pyproject.toml

```diff
@@ -53,6 +39,15 @@ enable_error_code = [
 module = ["pyarrow.*", "polars.polars", "matplotlib.*", "fsspec.*", "connectorx", "IPython.*"]
 ignore_missing_imports = true
 
+[[tool.mypy.overrides]]
+module = ["polars.*", "fsspec.*", "pyarrow.*"]
```

Comment:
I believe fsspec and pyarrow are not needed here.

Also, how exactly is adding `polars.*` here different from just including it in the general settings? I believe it's exactly the same (except for the fact that the override doesn't count for the `tests` folder).

Reply:
> I believe fsspec and pyarrow are not needed here.


Correct, after adding back the original missing imports, that is not needed anymore, so I have removed those.

> Also, how exactly is adding `polars.*` here different from just including it in the general settings? I believe it's exactly the same (except for the fact that the override doesn't count for the `tests` folder).

True, still more strict than turning off the warning altogether as it covers `tests` (although return-any is not very likely to encounter in a test suite I would think).

Reply:
Fair enough, we can keep it as an override!